From 381c47d1b37362fc886356eaa1f787d264bc2e1b Mon Sep 17 00:00:00 2001
From: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Date: Fri, 27 Sep 2019 12:44:26 -0700
Subject: [PATCH 0675/1851] dt-bindings: media: vpss-scaler: Add rev 2.2 to
 compatible string

commit 2ddb339f48661e03ee6aa96c557839e58e4a3fe3 from
https://github.com/Xilinx/linux-xlnx.git

This patch adds compatible string to support 2.2 version.

Signed-off-by: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,v-vpss-scaler.txt     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-scaler.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-scaler.txt
index c29b5f487dbf..05ca0cb33cad 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-scaler.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-scaler.txt
@@ -3,7 +3,7 @@
 
 Required properties:
 
-- compatible: Must be "xlnx,v-vpss-scaler-1.0".
+- compatible: Must be "xlnx,v-vpss-scaler-2.2" or "xlnx,v-vpss-scaler-1.0".
   The older string "xlnx,v-vpss-scaler" will be deprecated.
 
 - reg: Physical base address and length of the registers set for the device.
-- 
2.31.1

